SU-SWS/stanford_profile
# Upgrade drupal core to 9.3.0 with config updates

# READY FOR REVIEW

# Summary
- Upgrade drupal core to 9.3.0
- Exported all configs after database updates.

# Need Review By (Date)
- soon would be preferred.

# Urgency
- medium

# Steps to Test
1. Checkout the stack branch in https://github.com/SU-SWS/acsf-cardinalsites/pull/247/files
2. `composer install`
3. `blt di` or `blt drupal:update`
4. verify the site is still functional.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
